internal/plugin/terminal/ui.go

```diff
@@ -268,6 +268,10 @@ func (u *uiBridge) Close() error {
 	defer u.mu.Unlock()
 
 	err := u.evc.CloseSend()
+
+	// So see an EOF on the other side
+	u.evc.Recv()
```

Comment:
I would at least add a comment why we're ignoring the error here, seems like a thing a future eng might say "oh hey, we're missing error checking!"

Reply:
Sure.